torch/csrc/api/src/nn/modules/distance.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/nn/modules/distance.h>
+
+namespace F = torch::nn::functional;
+
+namespace torch {
+namespace nn {
+
+CosineSimilarityImpl::CosineSimilarityImpl(CosineSimilarityOptions options)
+    : options(options) {}
+
+void CosineSimilarityImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::CosineSimilarity()";
```

Comment:
We probably want to print all options args here as well. An example is `pretty_print` in torch/csrc/api/src/nn/modules/conv.cpp.